.github/workflows/linting.yml

```diff
@@ -45,12 +45,16 @@ jobs:
     steps:
       - name: Check out the repository
         uses: actions/checkout@v2
-
+      - name: Extract markdown from CSV.yaml
+        run: |-
+          sudo snap install yq && \
+          yq r config/manifests/bases/submariner.clusterserviceversion.yaml '.spec.description, .spec.links' > bundle.md
       - name: Run markdown-link-check
         uses: gaurav-nelson/github-action-markdown-link-check@v1
         with:
           config-file: ".markdownlinkcheck.json"
           check-modified-files-only: "yes"
+          file-path: "bundle.md"
```

Comment:
I think this will cause markdownlink-check to **only** check this file, instead of all the files **and** this file.

Reply:
@dfarrell07 according to [this documentation](https://github.com/gaurav-nelson/github-action-markdown-link-check) it says "specify additional files" but I think you are right I just saw the logs and it looks like it works only on this file. Confusing a little bit...